Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/02-NodeCreation/02-CreateNodeAggregateWithNode_ConstraintChecks_WithDimensions.feature

```diff
@@ -63,3 +63,22 @@ Feature: Create node aggregate with node
       | parentNodeAggregateId     | "sir-david-nodenborough"              |
       | originDimensionSpacePoint | {"language":"de"}                     |
     Then the last command should have thrown an exception of type "NodeAggregateDoesCurrentlyNotCoverDimensionSpacePoint"
+
+  Scenario: Try to create a node aggregate with a root parent and a sibling already claiming the name
+    # root nodes are special in that they have the empty DSP as origin, wich may affect constraint checks
+    When the command CreateNodeAggregateWithNode is executed with payload:
+      | Key                       | Value                                 |
+      | nodeAggregateId           | "sir-david-nodenborough"              |
+      | nodeTypeName              | "Neos.ContentRepository.Testing:Node" |
+      | parentNodeAggregateId     | "lady-eleonode-rootford"              |
+      | originDimensionSpacePoint | {"language":"de"}                     |
+      | nodeName                  | "document"                            |
+    And the graph projection is fully up to date
+    And the command CreateNodeAggregateWithNode is executed with payload and exceptions are caught:
+      | Key                       | Value                                 |
+      | nodeAggregateId           | "nody-mc-nodeface"                    |
+      | nodeTypeName              | "Neos.ContentRepository.Testing:Node" |
+      | parentNodeAggregateId     | "lady-eleonode-rootford"              |
+      | originDimensionSpacePoint | {"language":"de"}                     |
```

Comment:
shouldnt be that something different? But i checked it .. it doest matter the test fail before either way and now pass either way.

```suggestion
      | originDimensionSpacePoint | {"language":"mul"}                    |
```

Reply:
mul conflicts also in de, its specialization, thus this makes no difference. 
An additional test case would be create in mul, delete all specializations, then create in de. But that is out of scope here as it requires the deletion feature

Reply:
ups sorry i was on the wrong path i dont know what i thought ^^